### DISCUSSION TITLE
Run MBL in OpenModelica

### DISCUSSION THREAD

**Discussion by JordiGarciaCespedes** [lbl-srg/modelica-buildings#3250] [Q&A] [answered]
Hi!
I am new at Modelica and after learning the basics, I am trying to learn how to use your awesome Buildings library. To start with, I try to execute the thermofluid system exercise presented in one of your tutorial presentations (David Blum), but I found the first difficulties in defining the parameters of the components, since in OpenModelica, I cannot select the medium (water) in the paramaters dialog box, as it is shown for the Modelon Impact environment. Any clue of what I am doing wrong? is OpenModelica a suitable and tested environment for the library or maybe not...?
Thank you

**Comment by mwetter** [comment, marked as answer]
We are working on coverage of OpenModelica and OMEdit. Selecting the media is indeed not yet working, and I just filed an issue at https://github.com/OpenModelica/OpenModelica/issues/10248

**Reply by JordiGarciaCespedes** [reply]
Ok, thanks for answering :-)
I also managed to do the excercise "Simple ThermoFluid System" under OpenModelica environment (OMedit) after doing it in DYMOLA (free demo) and importing the code:
```modelica
model SimpleFluidModelDym
  Buildings.Fluid.MixingVolumes.MixingVolume vol(redeclare package Medium = Buildings.Media.Water,m_flow_nominal=1,V=2,nPorts=2);
  Buildings.Fluid.Sources.MassFlowSource_T Source(redeclare package Medium = Buildings.Media.Water,m_flow=1,T=303.15,nPorts=1);
  Buildings.Fluid.Sources.Boundary_pT Sink(redeclare package Medium = Buildings.Media.Water,T=293.15,nPorts=1);
equation 
  connect(Source.ports[1], vol.ports[1]);
  connect(Sink.ports[1], vol.ports[2]);
  annotation (Icon(coordinateSystem(preserveAspectRatio=false)),    Diagram(coordinateSystem(preserveAspectRatio=false)), uses(Buildings(version="9.1.0")));
end SimpleFluidModelDym;
```
As you say, the problem is related to redeclaring the packace Medium. In OMedit the parameter "Medium" does not appear in the dialog box of the components. It must be redeclared manually in the code.
Regards.
